### ISSUE TITLE
Crash if logging on and not running in terminal (v. 0.19.3)

### ISSUE THREAD

**Issue by mstawick** [getsentry/sentry-cli#19]
When run as background process (in my case by TeamCity server), sentry-cli will crash if it tries to log anything. Here's the trace:

```
[16:08:27][Step 1/1] thread '<unnamed>' panicked at 'called `Option::unwrap()` on a `None` value', ../src/libcore/option.rs:323
[16:08:27][Step 1/1] stack backtrace:
[16:08:27][Step 1/1]    1:     0x5574593ac2cf - std::sys::backtrace::tracing::imp::write::h22f199c1dbb72ba2
[16:08:27][Step 1/1]    2:     0x5574593b254d - std::panicking::default_hook::{{closure}}::h9a389c462b6a22dd
[16:08:27][Step 1/1]    3:     0x5574593b13e2 - std::panicking::default_hook::h852b4223c1c00c59
[16:08:27][Step 1/1]    4:     0x5574593b19d8 - std::panicking::rust_panic_with_hook::hcd9d05f53fa0dafc
[16:08:27][Step 1/1]    5:     0x5574593b1872 - std::panicking::begin_panic::hf6c488cee66e7f17
[16:08:27][Step 1/1]    6:     0x5574593b17b0 - std::panicking::begin_panic_fmt::hb0a7126ee57cdd27
[16:08:27][Step 1/1]    7:     0x5574593b1731 - rust_begin_unwind
[16:08:27][Step 1/1]    8:     0x5574593c03bf - core::panicking::panic_fmt::h9af671b78898cdba
[16:08:27][Step 1/1]    9:     0x5574593c02eb - core::panicking::panic::h1a2d1a6b50eaa468
[16:08:27][Step 1/1]   10:     0x5574591518d1 - <sentry_cli::utils::Logger as log::Log>::log::hb5d36d2002baf8ae
[16:08:27][Step 1/1]   11:     0x5574591a2b6f - log::__log::hf7539da7e3bbacdb
[16:08:27][Step 1/1]   12:     0x55745912d74d - sentry_cli::api::Api::request::hb5723a986d362fef
[16:08:27][Step 1/1]   13:     0x557459135342 - sentry_cli::api::Api::get_release::h2b5138dc5b5893a6
[16:08:27][Step 1/1]   14:     0x557459168a6f - sentry_cli::commands::releases::execute_files::h959219a5be183a90
[16:08:27][Step 1/1]   15:     0x55745916d1a3 - sentry_cli::commands::releases::execute::hac2cbb4d098e9e78
[16:08:27][Step 1/1]   16:     0x55745914b657 - sentry_cli::commands::execute::h8fa0a6bcf59d6768
[16:08:27][Step 1/1]   17:     0x55745914cd93 - sentry_cli::commands::main::h24f38387e1c1262a
[16:08:27][Step 1/1]   18:     0x5574590d4a51 - std::panicking::try::do_call::h6caed61cb4eba426
[16:08:27][Step 1/1]   19:     0x5574593ba016 - __rust_maybe_catch_panic
[16:08:27][Step 1/1]   20:     0x5574590e7671 - <F as alloc::boxed::FnBox<A>>::call_box::h2b5c562dae316ba8
[16:08:27][Step 1/1]   21:     0x5574593b0210 - std::sys::thread::Thread::new::thread_start::h50b05608a499d2b2
[16:08:27][Step 1/1]   22:     0x7efd3cba56f9 - start_thread
[16:08:27][Step 1/1]   23:     0x7efd3c6c5b5c - clone
[16:08:27][Step 1/1]   24:                0x0 - <unknown>
```

I worked it around by running sentry in detached screen and waiting for it to finish.

**Comment by mitsuhiko** [comment]
Which version of sentry-cli is this?

**Comment by mstawick** [comment]
Sorry, wrote it only in topic. It's v.0.19.3.

**Comment by mitsuhiko** [comment]
This should be fixed in 0.19.4 which is the current version.
